app/screens/edit_profile/components/email_field.tsx

```diff
@@ -15,7 +15,7 @@ import type {FloatingTextInputRef} from '@components/floating_text_input_label';
 const services: Record<string, string> = {
     gitlab: 'GitLab',
     google: 'Google Apps',
-    office365: 'Office 365',
+    office365: 'Entra ID',
```

Comment:
Why not change the prop name as well?

Reply:
the agreement was to keep all the ids, props, keys, etc to keep referencing as office365. Same happened in webapp and in server.